src/api/procedures/manageAllowedVenues.ts

```diff
@@ -0,0 +1,53 @@
+import { Asset, Procedure } from '~/internal';
+import { ManageAllowedVenuesParams, TxTags } from '~/types';
+import { ExtrinsicParams, ProcedureAuthorization, TransactionSpec } from '~/types/internal';
+
+/**
+ * @hidden
+ */
+export type Params = ManageAllowedVenuesParams;
```

Comment:
```suggestion
export type Params = ManageAllowedVenuesParams & {
    action: 'allow' | 'disallow';
    ticker: string;
};
```
Once you make the change for `ManageAllowedVenueParams` as mentioned in the other comment, you can change the above as suggested